src/Umbraco.Abstractions/Models/PublishedContent/HttpContextVariationContextAccessor.cs

```diff
@@ -7,22 +8,22 @@ namespace Umbraco.Web.Models.PublishedContent
     /// </summary>
     public class HttpContextVariationContextAccessor : IVariationContextAccessor
     {
-        public const string ContextKey = "Umbraco.Web.Models.PublishedContent.DefaultVariationContextAccessor";
-        public readonly IHttpContextAccessor HttpContextAccessor;
+        private readonly IRequestCache _requestCache;
+        private const string _contextKey = "Umbraco.Web.Models.PublishedContent.DefaultVariationContextAccessor";
```

Comment:
This is where naming conventions is confusing between r#, VS, etc... and I don't think we have a rule for this in our editorconfig - should constants always be ProperCase? Sames goes for static readonly field names, what convention should we follow? r# generally wants constants and static readonly to be ProperCase

Reply:
Looks like we can add 
```
dotnet_naming_symbols.private_fields.required_modifiers = abstract,async,readonly,static # all except const
```